using-timescaledb/adaptive-chunking.md

```diff
@@ -0,0 +1,123 @@
+# Adaptive chunking [](adaptive-chunking)
+
+The secret behind TimescaleDB's excellent performance with time-series
+data is its automatic partitioning scheme that splits the data across
+a number of N-dimensional *chunks*. However, the performance will only
+be good if the system achieves the "right" *chunk size* that won't
+blow up the memory budget. Oversized chunks result in slow INSERT
+performance when updating indexes and also negatively affect query
+speed.
+
+Unfortunately, the chunk size cannot be set directly as the size is
```

Comment:
I think this sentence is a bit confusing because some might think you can set the size, since you can modify the chunk interval. I think we should clarify here (and first paragraph) you are talking about chunk size _in bytes_. It's not needed everywhere in the document, but at least early on its useful so the reader knows what we mean by size.